README.md

```diff
@@ -0,0 +1,26 @@
+![Build Status](https://img.shields.io/github/workflow/status/omnivector-solutions/jobbergate/Test/main?label=main-build&logo=github&style=plastic)
```

Comment:
I noticed this needs to be fixed, following https://github.com/omnivector-solutions/jobbergate/pull/380

Reply:
Addressed in commit 2424f77